### ISSUE TITLE
Unable to process Http Request Get method

### ISSUE THREAD

**Issue by vivekTKV** [netty/netty#3558]
I am using the following class to access http request. netty/example/src/main/java/io/netty/example/portunification/PortUnificationServerHandler.java

while accessing the request
the request is in this format "ET /websocket HTTP/1.1"
expected "GET /websocket HTTP/1.1"

with that unable to process the http request.

**Comment by normanmaurer** [comment]
@vivekTKV which version ?

**Comment by vivekTKV** [comment]
in netty 4.0.26

**Comment by vivekTKV** [comment]
@normanmaurer 
I have identified the reason for the problem, if we put any system trace, the character is getting removed, I am not sure why it is happening.

In my code I did 
System.out.println("readByte::> "+in.readByte());

final int magic1 = in.getUnsignedByte(in.readerIndex());
final int magic2 = in.getUnsignedByte(in.readerIndex()+1);

so the http request is in the "ET /websocket HTTP/1.1" format, once the I removed the system trace it is working fine.

Thanks.
